src/main/java/com/gameList/gamelist/GameListController.java

```diff
@@ -30,22 +30,34 @@
 	
 	@PostConstruct
 	private void loadData() {
-		// Platforms
-		platformList = new ArrayList<Platform>();
-		
+		//Platforms
+		//Create Faculties
 		Platform platform1 = new Platform(1,"pc");
 		Platform platform2 = new Platform(2,"xbox");
 		Platform platform3 = new Platform(3,"playstation");
 		Platform platform4 = new Platform(4,"nintendo");
 		
+		//Create List
+		platformList = new ArrayList<Platform>();
+		
+		//Add to our List
+		platformList.add(platform1);
+		platformList.add(platform2);
+		platformList.add(platform3);
+		platformList.add(platform4);
+		
+		//Xbox
+		//Create Faculties
```

Comment:
I didn't find the part in the project that added the platform to the list, so I added it.